### ISSUE TITLE
documentation files missing

### ISSUE THREAD

**Issue by ssbarsky** [jordanbrown0/reg3#444]
Several documentation files are missing (when using the [?] button)
Cannot GET /doc/ExternalImport.html

Looking at the documentation in a separate browser log found these 404 errors (there may be others, haven't checked every branch.)

file:///C:/Users/sharon/Documents/Reg3/Boskone60/Documentation/ExternalImport.html
file:///C:/Users/sharon/Documents/Reg3/Boskone60/Documentation/ExternalExportManager.html
file:///C:/Users/sharon/Documents/Reg3/Boskone60/Documentation/ExternalExport.html

file:///C:/Users/sharon/Documents/Reg3/Boskone60/Documentation/StationManager.html
file:///C:/Users/sharon/Documents/Reg3/Boskone60/Documentation/ServerManager.html

file:///C:/Users/sharon/Documents/Reg3/Boskone60/Documentation/AdminPrinting.html
file:///C:/Users/sharon/Documents/Reg3/Boskone60/Documentation/AdminSync.html
file:///C:/Users/sharon/Documents/Reg3/Boskone60/Documentation/Zap.html
file:///C:/Users/sharon/Documents/Reg3/Boskone60/Documentation/ClientInstaller.html

**Comment by jordanbrown0** [comment]
Yes.  Every once in a while I get an itch and write another dozen or so.

**Comment by ssbarsky** [comment]
OK. I wasn't sure why it wasn't there, so I thought I would mention it.
Maybe the 404 page could be customized to say "coming soon".


Sharon

**Comment by jordanbrown0** [comment]
Sure.

I'll let you know when I think I have them all. At that point I'll want it to look like an error. Before that I'll look into what it would take to add a coming-soon.
